source/features/set-default-repositories-type-to-sources.tsx

```diff
@@ -10,13 +10,23 @@ function addSourceTypeToLink(link: HTMLAnchorElement): void {
 // No `include`, no `signal` necessary
 function init(): void {
 	observe([
-		'.header-nav-current-user ~ a[href$="tab=repositories"]', // "Your repositories" in the header profile dropdown
-		'[aria-label="User profile"] a[href$="tab=repositories"]', // "Repositories" tab on user profile
-		'[aria-label="Organization"] [data-tab-item="org-header-repositories-tab"] a', // "Repositories" tab on organization profile
+		`a:is([href^="/"], [href^="${location.origin}"])[href$="?tab=repositories"]`, // "Repositories" tab on user profile, only matches relative or same-origin links
+		`a:is([href^="/orgs/"], [href^="${location.origin}/orgs/"])[href$="/repositories"]`, // "Repositories" tab on organization profile, only matches relative or same-origin links
+		'[aria-label="Organization"] [data-tab-item="org-header-overview-tab"] a', // "Overview" tab on organization profile
+		'[aria-label="Organization"] a[data-tab-item="i0overview-tab"]', // "Overview" tab on organization profile (Global navigation update)
 		'a[data-hovercard-type="organization"]', // Organization name on repo header + organization list on user profile
 	], addSourceTypeToLink);
 }
 
 void features.add(import.meta.url, {
 	init,
 });
+
+/*
+
+Test URLs:
+
+- User profile: https://github.com/fregante
+- Organization profile: https://github.com/refined-github
```

Comment:
To avoid confusion: These aren't the only URLs on which the feature runs, it runs everywhere. Instead of "test URLs" we need a list of examples to test, e.g.

```suggestion
Live test:
- https://github.com/fregante?tab=repositories
- https://github.com/orgs/refined-github/repositories
- The "Your repositories" link in the user dropdown in the header
- The "Repositories" tab in
	- https://github.com/fregante
	- https://github.com/refined-github
```